frontend/src/components/StatusTransfer.tsx

```diff
@@ -111,4 +110,6 @@ const getTransferStatus = status => {
 
 }
 
+const getFormatAddress = address => address.slice(0, 12) + "..." + address.slice(-4)
```

Comment:
Also please add the link to [Voyager](https://voyager.online) or [Starkscan](https://starkscan.co).

Once you're finished with all the fixes, please re-request review.